src/esi/ExpatParser.cc

```diff
@@ -12,8 +12,28 @@
 
 #if USE_SQUID_ESI && HAVE_LIBEXPAT
 
+#include "base/RunnersRegistry.h"
 #include "esi/ExpatParser.h"
 
+static ESIParser::Register *prExpat = nullptr;
```

Comment:
Please make prExpat a (non-static) EsiExpatRr data member. We do not need to add this static and this global.

Reply:
Done in d28e46a